tracing-log/src/lib.rs

```diff
@@ -61,9 +61,30 @@ pub fn format_trace(record: &log::Record) -> io::Result<()> {
     let key = fields
         .field(&"message")
         .expect("log record fields must have a message");
+    let target_key = fields
+        .field(&"log.target")
+        .expect("log record fields must have a target");
+    let module_key = fields
+        .field(&"log.module_path")
+        .expect("log record fields must have a module_path");
+    let file_key = fields
+        .field(&"log.file")
+        .expect("log record fields must have a file");
+    let line_key = fields
+        .field(&"log.line")
```

Comment:
I know this is what the previous code already did, but it would be nice if we didn't have to call the `FieldSet::field` method every time, since it does a string comparison. We could probably optimize this by using a lazy static or something, and reusing the same key every time this method is called.

We could make this change in a follow-up PR, I just felt like it was worth mentioning.